code/modules/projectiles/ammo_datums.dm

```diff
@@ -385,8 +385,8 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	hud_state = "rifle"
 	hud_state_empty = "rifle_empty"
 	accurate_range = 15
-	damage = 30
-	penetration = 0
+	damage = 25
+	penetration = 10
```

Comment:
adding 10 penetration but removing 5 dmg doesn't feel like a big nerf.

Reply:
Most Xenos lack armor from what I hear, and the ones that do like Crusher and Defender on Fortify are quite OP anyways.